docker-compose.jsonnet

```diff
@@ -53,21 +56,27 @@
   // You can exec into the container and telnet to this port to get a python shell.
   backdoor_port:: 1234,
 
-  // Other nodes to backfill from. You should not include the local node.
+  // Other nodes to always backfill from. You should not include the local node.
   peers:: [
-    "http://wubloader.codegunner.com/"
   ],
 
   // Connection args for the database.
   // If database is defined in this config, host and port should be postgres:5432.
   db_args:: {
-    user: "postgres",
-    password: "postgres",
+    user: "vst",
+    password: "dbfh2019", // don't use default in production
```

Comment:
should note here that it must not contain `'` or `\`, since you're not doing any escaping in your setup script (nor do i think such escaping is worth bothering with)

this also applies to all the other password options.